pkg/volume/plugins.go

```diff
@@ -411,12 +411,13 @@ func NewPersistentVolumeRecyclerPodTemplate() *api.Pod {
 					VolumeSource: api.VolumeSource{},
 				},
 			},
+
 			Containers: []api.Container{
 				{
 					Name:    "pv-recycler",
 					Image:   "gcr.io/google_containers/busybox",
 					Command: []string{"/bin/sh"},
-					Args:    []string{"-c", "test -e /scrub && echo $(date) > /scrub/trash.txt && find /scrub -mindepth 1 -maxdepth 1 -delete && test -z \"$(ls -A /scrub)\" || exit 1"},
+					Args:    []string{"-c", "test -e /scrub && rm -rf /scrub/..?* /scrub/.[!.]* /scrub/* ; echo $(date) > /scrub/trash.txt && exit 0 "},
```

Comment:
I left a comment on #20495 about this scriptlet.  I understand this is needed to make the recycler work for RBD.  Better to remove it here, squash this PR, and then cherry pick the other commit from 20495.  When that merges, the cherry pick will fall out of this PR when you're hit with the rebase.